libs/designsystem/modal/src/modal-navigation.service.ts

```diff
@@ -340,4 +340,15 @@ export class ModalNavigationService {
     }
     return parentRoute;
   }
+
+  handleBackButton(modal: HTMLIonModalElement) {
+    const popStateEvent$ = fromEvent(window, 'popstate');
```

Comment:
Please inject `private windowRef: WindowRef` from `@kirbydesign/designsystem/types` in constructor and use this instead of global window object:
```suggestion
    const popStateEvent$ = fromEvent(this.windowRef, 'popstate');
```